snippets.json

```diff
@@ -6,7 +6,7 @@
   },
   "let": {
     "prefix": ["let"],
-    "body": ["let $0;", "in"],
+    "body": ["let $1;", "in $0"],
```

Comment:
Yes I mean writing the let part first. @frantisekhanzlikbl

Reply:
just so I'm absolutely sure, this change is correct, given that vscode moves the cursor to `$1` first and to `$0` second. right?

Reply:
Yes you are right, I forgot the context.